OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -867,7 +867,7 @@ void FindAndDeployBackupMcv(Actor self)
 
 			foreach (var mcv in mcvs)
 			{
-				if (mcv.IsMoving())
+				if (mcv.TraitsImplementing<IMove>().Any(m => m.IsMoving))
```

Comment:
All the other code assumes at most one IMove, so this could simplify down to a TraitOrDefault followed by a `!= null && .IsMoving`